share/config_global.toml

```diff
@@ -180,3 +180,19 @@ name = "Other"
         choices.ipv4 = "IPv4 Only"
         choices.ipv6 = "IPv6 Only"
         default = "both"
+
+    [misc.tls_passthrough]
+    name = "TLS-passthrough / SNI-based forwarding"
+    visible = "security_experimental_enabled"
+
+        [misc.tls_passthrough.tls_passthrough_enabled]
+        type = "boolean"
+        default = false
+
+        [misc.tls_passthrough.tls_passthrough_list]
+        type = "tags"
+        # Regex is just <domain regex>:<ip regex>
+        pattern.regexp = '^([^\W_A-Z]+([-]*[^\W_A-Z]+)*\.)+((xn--)?[^\W_]{2,}):((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)\.?\b){4}$'
```

Comment:
IMHO we should allow domain names (not only ipv4), and maybe port.

Reply:
cf https://github.com/YunoHost/yunohost/pull/1697#discussion_r1421757429, it's an experimental feature, the point is having something that works that we can iterate on, and in the nominal case there's an IPv4 available

Reply:
cf #1970